xbmc/cores/VideoPlayer/DVDSubtitles/SubtitlesStyle.h

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright (C) 2005-2021 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "utils/Color.h"
+
+#include <string>
+
+namespace KODI
+{
+namespace SUBTITLES
+{
+
+constexpr double SUBTITLE_VIEWPORT_HEIGHT = 1080.0;
+constexpr double SUBTITLE_VIEWPORT_WIDTH = 1920.0;
+
+enum class FontAlignment
+{
+  TOP_LEFT = 0,
+  TOP_CENTER,
+  TOP_RIGHT,
+  MIDDLE_LEFT,
+  MIDDLE_CENTER,
+  MIDDLE_RIGHT,
+  SUB_LEFT,
+  SUB_CENTER,
+  SUB_RIGHT
+};
+
+enum class FontStyle
+{
+  NORMAL = 0,
+  BOLD,
+  ITALIC,
+  BOLD_ITALIC
+};
+
+enum class BorderStyle
+{
+  OUTLINE,
+  OUTLINE_BOX
+};
+
+enum class AssOverrideStyles
+{
+  DISABLED = 0,
+  POSITIONS,
+  STYLES,
+  STYLES_POSITIONS
+};
+
+struct subtitlesStyle
```

Comment:
This is just a personal preference but maybe you can change the struct name just to style? The namespaces already identifies it as kodi::subtitles::style, makes it at least a bit more concise